src/lib/util/Util.ts

```diff
@@ -95,4 +99,17 @@ export class Util {
 	static pad(number: number): string {
 		return (number < 10 ? '0' : '') + number;
 	}
+
+	static async getResponse(value: string, interaction: { client: Client | GClient }) {
```

Comment:
```suggestion
	static async getResponse(value: string, interaction: Interaction) {
```

And then optionally `const client = interaction.client as GClient`

Reply:
No, because message commands and i don't want use `//@ts-expect-error`.